### PR TITLE
Enhance README instructions

### PR DESCRIPTION
**Pull request by LucBerge**
I struggled 3 hours before understanding the full process.
@gr2m

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "private": true,
   "description": "Probot & Google Cloud Functions example",
+  "homepage": "https://github.com//",
```
Comment (gr2m) [inline package.json:6]: This change seems unrelated

```suggestion
```

**Review by gr2m — APPROVED** [review]
I'm sorry to hear you struggled. Thank you so much for contributing back to make it easier for future users.

**Comment by LucBerge** [comment]
If the `homepage` field is not present in `package.json`, you will end up with the following issue https://github.com/probot/probot/issues/806. Try it yourself!
